apps/glusterfs/operations_block_volume.go

```diff
@@ -0,0 +1,353 @@
+//
+// Copyright (c) 2018 The heketi Authors
+//
+// This file is licensed to you under your choice of the GNU Lesser
+// General Public License, version 3 or any later version (LGPLv3 or
+// later), or the GNU General Public License, version 2 (GPLv2), in all
+// cases as published by the Free Software Foundation.
+//
+
+package glusterfs
+
+import (
+	"fmt"
+
+	"github.com/heketi/heketi/executors"
+	wdb "github.com/heketi/heketi/pkg/db"
+
+	"github.com/boltdb/bolt"
+)
+
+// BlockVolumeCreateOperation  implements the operation functions used to
+// create a new volume.
+type BlockVolumeCreateOperation struct {
+	OperationManager
+	noRetriesOperation
+	bvol *BlockVolumeEntry
+	//vol *VolumeEntry
```

Comment:
commented code

Reply:
This commit was a pure move and the comment existed previously. Will clean it up at a later time.